### PR TITLE
[WIP] Add localization tutorial

### PR DESCRIPTION
**Pull request by mrkvon**
_Update: Currently we seem to converge into using [weblate](https://hosted.weblate.org/projects/trustroots/). This tutorial will need to be rewritten, when the setup settles._

#### Proposed Changes

A tutorial for translating Trustroots. It will be valid when #1291 and #1289 get finished and merged. All the burden of extracting and converting translation files should lay on developers.

#### Testing Instructions

* Read the [tutorial](https://github.com/Trustroots/trustroots/blob/docs/localization/docs/l10n.md), try it out and propose improvements
* A feedback from non-technical perspective very welcome! Is it understandable? Does it sound reasonably easy to start translating?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ title: Trustroots Team Guide
 - [Angular Directives](Angular-Directives.md) (deprecated)
 - [App and server Monitoring](Monitoring.md)
 - [Database](Database.md)
-- [i18n](i18n.md)
+- [Internationalization (i18n)](i18n.md)
+- [Localization (l10n)](l10n.md)
```
Comment (simison) [inline docs/index.md]: The difference between the two might be confusing to some; how about adding "— for translators" and "— for developers" after each title?

Comment (mrkvon) [reply docs/index.md]: :+1: Done in 45c0144083fe7ff3e4e46f1f991b95c7a8f29875

File: docs/l10n.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Translating Trustroots (localization)
```
Comment (mrkvon) [inline docs/l10n.md]: I wrote this document to my best limited ability. Improvements and complete rewrites are very welcome!

**Comment by mrkvon** [comment]
out of date

**Pull request closed without merging**